docs/src/modules/components/DiamondSponsors.js

```diff
@@ -121,7 +121,7 @@ export default function DiamondSponsors() {
           />
         </StyledAnchor>
         <Link
-          aria-label={t('diamondSponsors')}
+          aria-label={`${t('becomeADiamondSponsor')}${t('diamondSponsorVacancies')}`}
```

Comment:
As far as I know, the aria-label makes no sense anymore since we now have the content on the page. Fixed in #40526.

(The aria-label was added in https://github.com/danilo-leal/material-ui/commit/0432934ec306458b6ff11e6572d1419fe2ab96e1#diff-b78e0f8daf77f3577d731d44ac96eaee7b085e3920b200b61952f5e1f3ea8c26R115)